src/Octoshift/Commands/CreateTeam/CreateTeamCommandBase.cs

```diff
@@ -17,7 +17,9 @@ public CreateTeamCommandBase() : base(name: "create-team", description: "Creates
     public virtual Option<string> TeamName { get; } = new("--team-name") { IsRequired = true };
 
     public virtual Option<string> IdpGroup { get; } = new("--idp-group");
-
+    {
+        Description = "The Identity Provider Group to link the team to. For Enterprise Managed Users only."
```

Comment:
there is an extra semi colon on line 19 that is a syntax error, once deleted the build should succeed.